doc/_extensions/zephyr/external_content.py

```diff
@@ -73,8 +73,8 @@ def adjust_includes(
     def _adjust(m):
         directive, fpath = m.groups()
 
-        # ignore absolute paths or existing files
-        if fpath.startswith("/") or (dstpath / fpath).exists():
+        # ignore absolute paths
+        if fpath.startswith("/"):
```

Comment:
Is this the right logic for windows style paths?

Why isn't it using os.path.isabs?

Reply:
These are Sphinx paths (they use / on all platforms)